### DISCUSSION TITLE
[Connector Request] Uxuy Connector

### DISCUSSION THREAD

**Discussion by lifecoder1988** [wevm/wagmi#4277] [Connector Request]
### What **novel use-case** does the Connector provide?

a wallet with it's own SDK (uxuyWallet) 

### Are the Connector's integrations production-ready and generally available?

We're building a connector that lets dApps run inside Telegram as mini-apps and, yep, they can call the UXUY Wallet mini-app too. It's all about seamless integration, and we think it could open up some interesting possibilities.

### Are you committed to actively maintaining the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by max-uxuy** [comment]
@tmm @jxom Can we please get an update on this? Thanks
